### DISCUSSION TITLE
Revoking Default Access for Keka

### DISCUSSION THREAD

**Discussion by hurlydan** [aonez/Keka#1422] [Q&A] [answered]
@aonez 

I'm on Ver. 1.3.7 and not very technical minded.
I'd like to revert to the default setting where I can choose the compression and extraction for each individual file
at the Finder level. 

Have read the instructions from this link:
https://github.com/aonez/Keka/wiki/Default-application

**Instructions (Pic1):**

<img width="920" alt="KH" src="https://github.com/aonez/Keka/assets/165161013/be031306-3d82-473b-8428-1ce08d8aa566">

**My Keka Helper Resources (Pic 2):** USERNAME]/Library/Containers/com.aone.keka/Data/Library/Application Support/Keka/

<img width="499" alt="KH1" src="https://github.com/aonez/Keka/assets/165161013/614914fb-e91c-4320-8d76-211b25725791">

The instructions say: "It is safe to remove it."
So do I just put the entire Keka folder highlighted in Pic 2 in the trash and empty it? Or just its contents, while leaving the folder
empty and intact?

Any restart or other steps required?

Many Thanks!

**Comment by aonez** [comment, marked as answer]
Sure! You can remove the Keka folder.

> I'd like to revert to the default setting where I can choose the compression and extraction for each individual file at the Finder level.

Note the helper does not have any functionality related to what you want here. What you want is to set Keka Preferences to ask always for a destination. Let me know if this is what you were looking for.

<img width="246" alt="Screenshot 2024-03-27 at 13 30 38" src="https://github.com/aonez/Keka/assets/480163/37aa4609-963a-4311-ba50-d9f6df70e10b">

<img width="736" alt="Screenshot 2024-03-27 at 13 28 26" src="https://github.com/aonez/Keka/assets/480163/16d21621-4f66-4b13-80d3-680b2fe8785e">

<img width="736" alt="Screenshot 2024-03-27 at 13 28 29" src="https://github.com/aonez/Keka/assets/480163/c151fd66-5904-46ee-8b5f-bc2e872c1f59">

**Comment by hurlydan** [comment]
Thanks! I'll try and delete that folder now. I trust the app must be fully quit while doing this.
And no restart of Mac required.

By selecting the default app for extraction, I meant highlighting: File>Get info and then selecting the default app
to perform extraction from there.

**Comment by aonez** [comment]
No need to close Keka or restart the Mac.

**Comment by hurlydan** [comment]
Update: the solution has worked. Thanks very much @aonez

**Reply by aonez** [reply]
Glad to help 👌
